### PR TITLE
Fix HDFS node for Integrated auth

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Fixes https://github.com/microsoft/azuredatastudio/issues/12905 - the casing was different so the check wasn't being done properly.

### CODE REVIEW

File: extensions/mssql/src/sqlClusterLookUp.ts [hunk 1]
```diff
@@ -149,7 +149,7 @@ async function createSqlClusterConnInfo(sqlConnInfo: azdata.IConnectionProfile |
 
 async function getClusterController(controllerEndpoint: string, connInfo: ConnectionParam): Promise<bdc.IClusterController | undefined> {
 	const bdcApi = <bdc.IExtension>await vscode.extensions.getExtension(bdc.constants.extensionName).activate();
-	let authType: bdc.AuthType = connInfo.options[constants.authenticationTypePropName] === AuthType.Integrated ? 'integrated' : 'basic';
+	let authType: bdc.AuthType = connInfo.options[constants.authenticationTypePropName].toLowerCase() === AuthType.Integrated ? 'integrated' : 'basic';
```
Comment (alanrenmsft) [inline extensions/mssql/src/sqlClusterLookUp.ts:152]: this should be a const, right?

**Review by alanrenmsft — APPROVED** [review]

File: extensions/mssql/src/sqlClusterLookUp.ts (hunk 1 above)
Comment (Charles-Gagnon) [reply extensions/mssql/src/sqlClusterLookUp.ts:152]: The 'integrated' and 'basic' values? Not needed, bdc.AuthType is `'integrated' | 'basic'` type

Comment (alanrenmsft) [reply extensions/mssql/src/sqlClusterLookUp.ts:152]: I mean the authType variable

Comment (Charles-Gagnon) [reply extensions/mssql/src/sqlClusterLookUp.ts:152]: Ah, yeah it could be

**Pull request merged**